src/redux/actions/moviesAction.ts

```diff
@@ -0,0 +1,57 @@
+import { AppDispatch } from "../../types/redux.interface";
+import Movie from "../../types/movie.interface";
+import { GET_MOVIES_ERROR, GET_MOVIES_START, GET_MOVIES_SUCCESS } from "../types";
+
+const BACKEND_URL = "http://localhost:4000/movies";
+const MOVIES_PER_PAGE = "9";
```

Comment:
I guess it can be number - it will be less confusing :)

Reply:
If I don't declare MOVIES_PER_PAGE as a string. I will obtain a type error from the typescript validation